src/components/UserCollective.js

```diff
@@ -74,43 +74,45 @@ class UserCollective extends React.Component {
       },
       'organization.collective.edit': {
         id: 'organization.collective.edit',
-        defaultMessage: 'edit organization',
+        defaultMessage: 'edit Organization',
       },
       'user.collective.edit': {
         id: 'user.collective.edit',
         defaultMessage: 'edit profile',
       },
       'user.collective.memberOf.collective.host.title': {
         id: 'user.collective.memberOf.collective.host.title',
-        defaultMessage: "I'm hosting {n, plural, one {this collective} other {these {n} collectives}}",
+        defaultMessage: "I'm hosting {n, plural, one {this Collective} other {these {n} Collectives}}",
       },
       'user.collective.memberOf.organization.admin.title': {
         id: 'user.collective.memberOf.organization.admin.title',
-        defaultMessage: "I'm an administrator of {n, plural, one {this organization} other {these {n} organizations}}",
+        defaultMessage: "I'm an admin of {n, plural, one {this Organization} other {these {n} Organizations}}",
       },
       'user.collective.memberOf.organization.member.title': {
         id: 'user.collective.memberOf.organization.member.title',
-        defaultMessage: "I'm a member of {n, plural, one {this organization} other {these {n} organizations}}",
+        defaultMessage: "I'm a team member of {n, plural, one {this Organization} other {these {n} Organizations}}",
       },
       'user.collective.memberOf.collective.admin.title': {
         id: 'user.collective.memberOf.collective.admin.title',
-        defaultMessage: "I'm a core contributor of {n, plural, one {this collective} other {these {n} collectives}}",
```

Comment:
Why not just admin?

Reply:
We decided to call them core contributors and just indicate with a little star or something if they are also admins, because "contributor" is the central identity word we are focusing on. But I don't think it's that important in this context so if you want to use "Collective Admin" instead that's OK.